LOADERS/REDIS_SYSTEM/REDIS_LOAD/REDIS_LOAD.py

```diff
@@ -11,10 +11,22 @@
 
 
 @flojoy
-def LINE(v, params):
+def REDIS_LOAD(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """The REDIS_LOAD node loads data directly from REDIS.
+
+    Parameters
+    ----------
+    referred_node: list of str
+        The node where REDIS data will be loaded from.
+
+    Returns
+    -------
+    dataframe
+        The dataframe loaded from Redis. Ordered pair.
+    """
     referred_node = params["referred_node"]
-    x = v[0].y
-    if referred_node is not "":
+    x = dc_inputs[0].y
+    if referred_node != "":
         referred_node_key = referred_node.split("-")[0]
         try:
             job = Job.fetch(
```

Comment:
Node id's are used as job id when they are enqueued to rq. So It's unnecessary to fetch the job from RQ just to get job id. Instead value passed to referred_node can be used as job_id